src/Behat/Behat/Context/Cli/InteractiveContextIdentifier.php

```diff
@@ -57,6 +57,10 @@ public function __construct(TranslatorInterface $translator, InputInterface $inp
      */
     public function guessTargetContextClass(ContextEnvironment $environment)
     {
+        if ($this->input->hasParameterOption('--no-interaction')) {
```

Comment:
I did [initially](https://github.com/Behat/Behat/pull/955/commits/283586a1ae521e5be0ef77c8042b18ec141caad6#diff-cc5e27e0868da5db12e6f72502938043R58). It breaks apart in the `Symfony\Console` prior to 2.7